api/src/main/java/com/capitalone/dashboard/config/LoggingFilter.java

```diff
@@ -108,6 +108,13 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             }
 
         } else {
+            //new code added 
+			//adds headers to response to allow CORS	
+			httpServletResponse.addHeader("Access-Control-Allow-Origin", "*");
```

Comment:
Allowing arbitrary callers is a potential security vulnerability. It should be restricted to only those that are needed.